WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/BlogDashboardCardFrameView.swift

```diff
@@ -58,7 +58,7 @@ class BlogDashboardCardFrameView: UIView {
 
     /// Ellipsis Button displayed on the top right corner of the view.
     /// Displayed only when an associated action is set
-    private lazy var ellipsisButton: UIButton = {
+    private(set) lazy var ellipsisButton: UIButton = {
```

Comment:
[nitpick] Since `ellipsisButton` is now public, we can now remove `ellipsisButtonFrame` and instead use the `ellipsisButton.frame` directly

Reply:
This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/18290/commits/399b0156b731675903cc99f97ba9acf0adcd2a04. 🙂